### PR TITLE
Hardware

### PR DESCRIPTION
**Pull request by alexandra214**
### Pull Request Overview

This pull request adds the documentation for the Clock-in project.

### TODO or Help Wanted

Please let me know if there is anything to change.

### Author

Signed-off-by: Oblu Alexandra Mihaela oblu.alexandra@proton.me

### CODE REVIEW

File: docs/project/2024/alexandra.oblu/alexandra.oblu.md
Comment (alexandruradovici) [inline docs/project/2024/alexandra.oblu/alexandra.oblu.md:1]: Please edit index.md instead of adding a new file.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/alexandra.oblu/alexandra.oblu.md
Comment (alexandra214) [reply docs/project/2024/alexandra.oblu/alexandra.oblu.md:1]: I fixed it.

File: docs/project/2024/alexandra.oblu/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ A clock-in machine used to keep track of attendance.
 :::info
 
 **Author**: Oblu Alexandra \
-**GitHub Project Link**: [link_to_github](https://github.com/UPB-FILS-MA/project-alexandra214)
+**GitHub Project Link**: [Clock-in](https://github.com/UPB-FILS-MA/project-alexandra214)
```
Comment (alexandruradovici) [inline docs/project/2024/alexandra.oblu/index.md:9]:
```suggestion
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-alexandra214
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/alexandra.oblu/index.md (hunk 1 above)
Comment (alexandra214) [reply docs/project/2024/alexandra.oblu/index.md]: I fixed it.

**Review by alexandruradovici — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**